single_tech_samples/databricks/sample4_ci_cd/devops/templates/jobs/deploy-libs-job.yml

```diff
@@ -0,0 +1,62 @@
+parameters:
+- name: environmentName
+  type: string
+- name: databricksDomain
+  type: string
+- name: databricksToken
+  type: string
+- name: databricksNotebookPath
+  type: string
+- name: databricksClusterId
+  type: string
+
+jobs:
+- deployment: deploy_libs
+  environment: ${{ parameters.environmentName }}
+  displayName: 'Deploy libs to Databricks'
+  pool:
+    vmImage: 'ubuntu-latest'
+  variables:
+    pythonVersion: 3.8
+  strategy:
+    runOnce:
+      deploy:
+        steps:
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '$(pythonVersion)'
+            addToPath: true
+            architecture: 'x64'
+          displayName: 'Use Python Version: $(pythonVersion)'
+
+        - script: |
+            python -m pip install --upgrade pip
+            pip install databricks-cli
+          displayName: 'Setup Agent'
+        - script: |
+            echo "Uploading libs at ${LIB_PATH} to workspace (${DATABRICKS_LIB_PATH})..."
```

Comment:
You print out `"Uploading libs at ${LIB_PATH} to workspace (${DATABRICKS_LIB_PATH})..."`
But you actually upload to `dbfs:/FileStore/jars` :)

Reply:
@devlace Changed it to use ${DATABRICKS_LIB_PATH}. Thanks :).